config/warble.rb

```diff
@@ -0,0 +1,179 @@
+# Disable Rake-environment-task framework detection by uncommenting/setting to false
+# Warbler.framework_detection = false
+
+# Warbler web application assembly configuration file
+Warbler::Config.new do |config|
+  # Features: additional options controlling how the jar is built.
+  # Currently the following features are supported:
+  # - gemjar: package the gem repository in a jar file in WEB-INF/lib
+  # - executable: embed a web server and make the war executable
+  # - compiled: compile .rb files to .class files
+  # config.features = %w(gemjar)
+
+  # Application directories to be included in the webapp.
+#  # config.dirs = %w(app config db lib log script vendor tmp)
+  config.dirs = %w(server/config server/local server/log server/public server/test-files UI)
+
+  # Additional files/directories to include, above those in config.dirs
+  # config.includes = FileList["db"]
+  config.includes = FileList["server/*.rb"]
```

Comment:
Not all files in the directory will be included and the documentation for config.dirs only discusses including full directories.